packages/nimble-components/src/dialog/tests/dialog-matrix.stories.ts

```diff
@@ -68,3 +68,13 @@ export const dialogDarkThemeBlackBackground: Story = createFixedThemeStory(
 );
 
 dialogDarkThemeBlackBackground.play = playFunction;
+*/
+
```

Comment:
Could probably just update the play function to check if dialog is supported. Maybe
```ts
const playFunction = (): void => {
   if (window.HTMLDialogElement) {
        void document.querySelector('nimble-dialog')!.show();
   }
};
```

or
```ts
const hasDialog = document.createElement('dialog').showModal !== undefined;
const playFunction = (): void => {
   if (hasDialog) {
        void document.querySelector('nimble-dialog')!.show();
   }
};
```